exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/ElasticsearchExporterTest.java

```diff
@@ -45,6 +48,18 @@ public void setUp() {
     esClient = mockElasticsearchClient();
   }
 
+  @Test
+  public void shouldNotFailOnOpenIfElasticIsUnreachable() {
+    // given
+    final ElasticsearchClient client = spy(new ElasticsearchClient(config, new ZbLogger("test")));
+    final ElasticsearchExporter exporter = createExporter(client);
+    config.index.createTemplate = true;
+
+    // when - then : only fails when trying to export, not before
+    openExporter(exporter);
+    assertThatThrownBy(testHarness::export).isNotNull();
```

Comment:
Here we could check for `ElasticsearchExporterException` right?

Reply:
In theory yes, if another error is thrown then you could argue the test isn't testing the correct thing...but on the other hand, if an error is thrown it's also unexpected and it's a bug, so...

But yeah I can add it, it costs nothing to do so.